.github/workflows/bench_pr.yml

```diff
@@ -10,6 +10,10 @@ on:
 
   workflow_dispatch:
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
```

Comment:
Sometimes I want to get benchmarks manually, does this prevent that?

Reply:
No, it just cancels previous pending/in-progress runs. Doesn't prevent from manually triggering flows.